install/kubernetes/helm/istio/charts/meshexpansion/templates/config-envoy.yaml

```diff
@@ -0,0 +1,181 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: meshexpansion-envoy-config
+  namespace: istio-system
```

Comment:
Why can’t we do the same thing as the gateways ? The istio-system namespace can be dynamically obtained from helm stuff

Reply:
Done